src/Perlang.Tests/ConsoleApp/ProgramTest.cs

```diff
@@ -1,481 +0,0 @@
-
-                StderrContent.Should().BeEmpty();
-
-                // There used to be a bug with the reporting of both of these warnings and errors, where the line number
-                // was messed up (#276)
-                StdoutLines.Should().Contain(
-                        "[line 6] Warning at 'greet': Null parameter detected for 'name'")
-                    .And.Contain(
-                        "[line 2] Operands must be numbers, not AsciiString and null"
-                    );
-
-                exitCode.Should().Be((int)Program.ExitCodes.RUNTIME_ERROR);
-            }
-
-            [SkippableFact(DisplayName = "with -Wno-error=null-usage parameter: can be combined with script argument")]
-            public void with_no_error_null_usage_parameter_can_be_combined_with_script_argument()
-            {
-                Skip.If(PerlangMode.ExperimentalCompilation, "Not supported in compiled mode");
-
-                int exitCode = Program.MainWithCustomConsole(new[] { "-Wno-error", "null-usage", "test/fixtures/argv_pop.per", "hello, world" }, testConsole);
-
-                StderrContent.Should().BeEmpty();
-
-                StdoutContent.Should().Contain(
-                    "hello, world"
-                );
-
-                exitCode.Should().Be(0);
-            }
-
-            [Fact(Skip = "Eval option is temporarily going away: https://github.com/perlang-org/perlang/issues/406")]
-            public void without_parameter_throws_error_on_null_assignment()
-            {
-                Program.MainWithCustomConsole(new[] { "-e", "var s: string; s = null; print 10;" }, testConsole);
-
-                StdoutContent.Should().Contain(
-                    "Error at 's': Null assignment detected"
-                );
-
-                // This test ensures that the 'print 10' never gets executed. When a compiler _error_ occurs, the
-                // program should not be executed.
-                StdoutContent.Should().NotContain("10");
-
-                StderrContent.Should().BeEmpty();
-            }
-
-            [Fact(Skip = "Eval option is temporarily going away: https://github.com/perlang-org/perlang/issues/406")]
-            public void without_parameter_throws_error_when_initializing_to_null()
-            {
-                Program.MainWithCustomConsole(new[] { "-e", "var s: string = null; print 10;" }, testConsole);
-
-                StdoutContent.Should().Contain(
-                    "Error at 's': Initializing variable to null detected"
-                );
-
-                // This test ensures that the 'print 10' never gets executed. When a compiler _error_ occurs, program
-                // execution should be aborted.
-                StdoutContent.Should().NotContain("10");
-
-                StderrContent.Should().BeEmpty();
-            }
-        }
-    }
-}
```

Comment:
Losing this whole test is the price we have to pay for removing the `IPerlangConsole` and related code. :cry: We'll have to live with it for now.